db/post_migrate/20191107190330_remove_suppress_from_latest_from_category.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class RemoveSuppressFromLatestFromCategory < ActiveRecord::Migration[6.0]
+  DROPPED_COLUMNS ||= {
+    categories: %i{suppress_from_latest}
+  }
+
+  def up
+    ids = DB.query_single("SELECT id::text FROM categories WHERE suppress_from_latest = TRUE")
+
+    if ids.present?
+      muted_ids = DB.query_single("SELECT value from site_settings WHERE name = 'default_categories_muted'").first
+      ids += muted_ids.split("|") if muted_ids.present?
+      ids.uniq!
+
+      # We shouldn't encourage to have more than 10 categories in `default_categories_muted` site setting.
+      if ids.count <= 10
+        DB.exec(<<~SQL, muted: CategoryUser.notification_levels[:muted])
+          INSERT INTO category_users (category_id, user_id, notification_level)
+            SELECT c.id category_id, u.id user_id, :muted
+            FROM users u
+              CROSS JOIN categories c
```

Comment:
I hope this `CROSS JOIN` won't hit any performance issues. Anyway, we're not migrating more than 10 categories.